src/actions/get-bounty-closed-event.ts

```diff
@@ -13,27 +13,64 @@ import {Push} from "../services/analytics/push";
 import {AnalyticEventName} from "../services/analytics/types/events";
 import updateSeoCardBounty from "src/modules/handle-seo-card";
 import { savePointEvent } from "../modules/points-system/save-point-event";
+import calculateDistributedAmounts from "src/modules/calculate-distributed-amounts";
+import { Proposal } from "@taikai/dappkit";
+import { networksAttributes } from "src/db/models/networks";
 
 export const name = "getBountyClosedEvents";
 export const schedule = "*/12 * * * *";
 export const description = "Move to 'Closed' status the bounty";
 export const author = "clarkjoao";
 
-async function updateUserPayments(proposal, transactionHash, issueId, tokenAmount) {
-  return Promise.all(
-    proposal.details.map(async (detail) => {
+async function updateUserPayments(proposal: Proposal, 
+                                  transactionHash: string, 
+                                  issueId: number, 
+                                  tokenAmount: string | number, 
+                                  chainId: number, 
+                                  network: networksAttributes, 
+                                  merger: string) {
+  const chain = await db.chains.findOne({
+    where: {
+      chainId: chainId,
+    }
+  });
+
+  if (!chain?.closeFeePercentage || !network.mergeCreatorFeeShare || !network.proposerFeeShare) {
+    logger.warn(`war not possible to save payments for task ${issueId} because chain ${chainId} was not found in database`);
```

Comment:
typo "war" -> "was"
also, error message does not describe the condition, 
`Not possible to save payment for ${issueId} as it's missing closeFeePercentage, mergeCreatorFeeShare or ProposerFeeShare` would be more informative